tiered-storage/file-system/src/main/java/org/apache/bookkeeper/mledger/offload/filesystem/impl/FileStoreBackedReadHandleImpl.java

```diff
@@ -113,7 +122,10 @@ public LedgerMetadata getLedgerMetadata() {
                 key.set(nextExpectedId - 1);
                 reader.seek(key);
                 while (entriesToRead > 0) {
+                    long startReadTime = System.nanoTime();
                     reader.next(key, value);
+                    mxBean.recordReadOffloadDataLatency(managedLedgerName, System.nanoTime() - startReadTime,
```

Comment:
Maybe we need to add try-catch for the `reader.next`? It may throw IOException which would miss the latency calculation?

Reply:
I think we don't need a try-catch block here, if there are a exception, record latency is meaningless